### DISCUSSION TITLE
Welcome to VidCoder Discussions!

### DISCUSSION THREAD

**Discussion by RandomEngy** [RandomEngy/VidCoder#722] [General]
If you want to discuss something or ask a question that isn't an issue, this is the place!

**Comment by 55trucker** [comment]
Any idea when the program will be updated to include x264 core 160? 
I've come across 160 in another encoding program,

to the best of your knowledge are there any actual *core encoding* improvements with every new version that is introduced?

for example....157 vs the older148

thx

**Comment by RandomEngy** [comment]
I pick up new builds from HandBrake with Beta releases. If HandBrake has updated their core to use it, it will come into VidCoder on one of those updates.

**Comment by Rayman24365** [comment]
In the 6.20 Beta you made the adding order alphabetical by default. Can't you make that a switch?
I often add files in order of size or date, sometimes even by extension and if I can't do that anymore it will hinder me greatly in doing what I want to do. I don't understand why the adding order has to be overruled. Cheers.

**Reply by RandomEngy** [reply]
I added that switch in 6.21 Beta under Global Options -> Advanced.
